core-bundle/contao/classes/DataContainer.php

```diff
@@ -785,9 +785,9 @@ protected function row($strPalette=null)
 	 *
 	 * @return string
 	 */
-	public function help($strClass='')
+	public function help($strClass='', $strDescription=null)
 	{
-		$return = $GLOBALS['TL_DCA'][$this->strTable]['fields'][$this->strField]['label'][1] ?? null;
+		$return = $strDescription ?? $GLOBALS['TL_DCA'][$this->strTable]['fields'][$this->strField]['label'][1] ?? null;
```

Comment:
This is a new concept that did not exist before. Normally, you would dynamically overwrite `$GLOBALS['TL_DCA'][$this->strTable]['fields'][$this->strField]['label']` in a listener.

@contao/developers Do you agree with this concept?

Reply:
Maybe @aschempp can shed some light on why this is needed and what other use cases this would simplify. There must be a reason for his decision here.